### DISCUSSION TITLE
Error downloading installation file. Read timed out.

### DISCUSSION THREAD

**Discussion by acataranciuc15-mm** [dbeaver/dbeaver#20541] [General]
Due to network configuration connection to dbeaver site to get new update files takes longer than normal. Which is why my auto upgrade fails on network timeout error. Please advise how can I increase this timeout to allow enough time for the file to be found and downloaded?

Error message: "Error downloading installation file. Read timed out."
Screenshot:
![image](https://github.com/dbeaver/dbeaver/assets/91484539/bfbf22fc-87e2-49dc-b91e-410171163a1d)

**Comment by E1izabeth** [comment]
You could try downloading installation file from the website manually through the web browser and then run it.

By the way, we already have feature request about that #18457. You can vote for the issue with 👍🏻 under the issue description.

**Reply by acataranciuc15-mm** [reply]
Hello Elizabeth,, yes that is the way how I update my DBeaver, but after more than a year its becoming annoying. I would prefer to get the resolution for increasing the timeout. I upvoted the case you've shared, and will wait for the solution there. Thanks

**Comment by struck89** [comment]
Hi @E1izabeth 
I don't think this is a real time out issue. I have the same error. In my case I am using ZScaler, the file downloads fine until 99%, then the error shows up. The whole thing is less than 30seconds. For some reason DBeaver doesn't like a file downloaded with ZScaler on. So I just tried to add the ZScaler root certificate into cacerts file, but didn't work. I have to turn off ZScaler (which my company doesn't like) for the download & install to succeed

I added the cacert like this:
`PS C:\Users\[...]\AppData\Local\DBeaver\jre\bin> .\keytool.exe -import -trustcacerts -alias zscaler -file 'C:\Users\[...]\Documents\cacert\ZscalerRootCertificate-2048-SHA256.pem' -keystore 'C:\Users\[...]\AppData\Local\DBeaver\jre\lib\security\cacerts'`

With the password being "changeit" as suggested by ChatGPT haha
